src/ctap/mod.rs

```diff
@@ -765,9 +765,12 @@ impl CtapState {
 
         let mut cred_protect_policy = extensions.cred_protect;
         if cred_protect_policy.unwrap_or(CredentialProtectionPolicy::UserVerificationOptional)
-            < DEFAULT_CRED_PROTECT.unwrap_or(CredentialProtectionPolicy::UserVerificationOptional)
+            < env
+                .customization()
+                .default_cred_protect()
```

Comment:
Let's maybe introduce a local variable for those 2 usages.

Reply:
Done